create_collections.ps1

```diff
@@ -0,0 +1,30 @@
+$collections = Get-Content -Path ./collections.json | ConvertFrom-Json
+
+$cosmosAccount = "dfc-dev-prov-cdb"
+$cosmosResourceGroup = "dfc-dev-prov-rg"
+
+foreach($collection in $collections) {
+    Write-Host "Creating collection: $($collection.Name)"
+    $params = @(
+        "cosmosdb"
+        "collection"
+        "create"
+        "--db-name"
+        "providerportal"
+        "--name"
+        $cosmosAccount
+        "--resource-group-name"
+        $cosmosResourceGroup
+        "--collection-name"
+        $collection.Name
+        "--throughput"
+        $collection.RequestUnits
+    )
+
+    if($collection.PartitionKey) {
+        $params += "--partition-key-path"
+        $params += $collection.PartitionKey
+    }
+
+    & az $params | Out-Null
```

Comment:
We use the CosmosDb PS module for managing Cosmos (https://github.com/PlagueHO/CosmosDB), is there a reason this wasn't used in this case?  (if this is a one off job then not worth spending time rewriting)

Reply:
This is pretty a one-off script for supporting next week's go-live for Course Directory